graphql_query_examples/graphql_mutation_examples.md

```diff
@@ -28,6 +28,32 @@ mutation aliasCreate_for_a_service{
 }
 ```
 
+### 🧬 aliasDelete for a service, get services first
+
+```graphql
+query get_services{
+  account{
+    services{
+      nodes{
+        name
+        id
+        aliases
```

Comment:
```suggestion
        managedAliases
```

I highly recommend using `managedAliases` wherever possible. `aliases` includes the slug of a service, which can never be changed or deleted by a user. Since the example below is to delete an alias, the user should query `managedAliases` which is a list of every single alias that can actually be managed by a user.

Just a note, `managedAliases` isn't available on most resources, it was added on services because of the slug issue.

Reply:
TIL we have a `managedAliases` field on some of our resources 👍

Reply:
Hey this is awesome! I am inclined to keep both, since only using `managedAliases` will hide/obscure the "locked alias" aka the slug. I can add a comment in any query/mutation that includes `managedAliases` that these are actual aliases that can be managed by a user.